.github/scripts/config.sh

```diff
@@ -68,8 +68,8 @@ DEJAGNU_FILE=${DEJAGNU_FILE:-$ROOT_PATH/.github/scripts/toolchain/site.exp}
 OPENBLAS_REPO=${OPENBLAS_REPO:-OpenMathLib/OpenBLAS}
 OPENBLAS_BRANCH=${OPENBLAS_BRANCH:-develop}
 
-ZLIB_REPO=${ZLIB_REPO:-madler/zlib}
-ZLIB_BRANCH=${ZLIB_BRANCH:-develop}
```

Comment:
Why not using `master` branch of the upstream repository? Should be stable enough.